rbac_api_with_domains_test.go

```diff
@@ -250,3 +250,14 @@ func TestDeleteAllUsersByDomain(t *testing.T) {
 		{"alice", "admin", "domain1"},
 	})
 }
+
+// testGetAllDomains tests GetAllDomains()
+func testGetAllDomains(t *testing.T, e *Enforcer, res []string) {
```

Comment:
Are there some tests called this func?

Reply:
I've added the test to call this function.